.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Lets use https://www.toptal.com/developers/gitignore/api/node. you can copy the gitignore this generates